engine/apps/slack/models/slack_team_identity.py

```diff
@@ -146,8 +146,8 @@ def get_users_from_slack_conversation_for_organization(self, channel_id, organiz
         members = self.get_conversation_members(sc, channel_id)
 
         return organization.users.filter(
+            User.build_permissions_query(RBACPermission.Permissions.CHATOPS_WRITE, organization),
```

Comment:
**NOTE**: `User.build_permissions_query` now returns `django.db.models.Q` object(s) instead of a dict